src/screens/AirdropDashboard/components/AirdropInProgress.tsx

```diff
@@ -98,7 +98,9 @@ export const AirdropInProgress: FC<Props> = props => {
     <>
       <View style={styles.infoContainer}>
         <Text style={typography.headline4}>{i18n.airdrop.title}</Text>
-        <Text style={[styles.description, styles.spaceTop]}>{i18n.airdrop.dashboard.desc1}</Text>
+        <Text style={[styles.description, styles.spaceTop]}>
+          {props.subscribedWallets?.length > 0 ? i18n.airdrop.dashboard.desc1WithWallets : i18n.airdrop.dashboard.desc1}
```

Comment:
AirdropInProgressContent should be a separate component, can be in this file, because now it's created on every rerender

Reply:
done